Gemfile

```diff
@@ -24,7 +24,9 @@ gem 'dough-ruby',
 gem 'jquery-rails'
 gem 'kaminari'
 gem 'letter_opener', group: :development
-gem 'mas-rad_core', '0.0.54'
+gem 'mas-rad_core',
+    github: 'moneyadviceservice/mas-rad_core',
+    branch: 'feature/add_trading_names'
```

Comment:
It has, but not released on RubyGems yet. I usually leave it til the pre-merge squash as things can end up getting raised in review that necessitate further changes to the gem.